src/GAP/iso_oscar_gap.jl

```diff
@@ -70,9 +70,12 @@ function _iso_oscar_gap_field_finite_functions(FO::Union{Nemo.fpField, Nemo.FpFi
    return (f, finv)
 end
 
+Nemo.lift(R::ZZRing, x::ZZRingElem) = x
+Nemo.lift(R::ZZRing, x::Integer) = ZZRingElem(x)
```

Comment:
I think having these would be useful in Nemo in general? @fieker @thofma ?

But if these are undesirable, I can revise the fix here to instead properly distinguish based on the type of `FO` (I plan to do that anyway). But this takes more time, and for now I just want to fix the CI as quickly as possible.

Reply:
Hm, I am not a big fan. This is all just coercion, for which we do not use the word `lift`.